api/google-search/src/main/java/com/google/search/googlesearch/models/SearchAutoCompleteModel.java

```diff
@@ -0,0 +1,23 @@
+package com.google.search.googlesearch.models;
+
+import java.util.ArrayList;
+
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
Do not annotate models with @Component.

Reply:
Included javax.inject package in pom.xml and replaced @Component with @Named